### PR TITLE
chore: dynamic plugins are all now expressed as wrappers (RHIDP-5103)

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: dynamic plugins are all now expressed as wrappers so we don't need to parse janus-plugins repo anymore

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/RHIDP-5103

for 1.4: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/754
for main: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/753

### How to test this PR?
N/A

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] Code produced is complete
- [ ] Code builds without errors
- [ ] Tests are covering the bugfix
- [ ] Relevant user documentation updated
- [ ] Relevant contributing documentation updated

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: modules/dynamic-plugins/ref-community-plugins.adoc [hunk 1]
```diff
@@ -10,267 +10,10 @@
...
-
-`TECH_RADAR_DATA_URL`
-
-
-|Utils  |`https://npmjs.com/package/@roadiehq/scaffolder-backend-module-utils/v/2.0.2[@roadiehq/scaffolder-backend-module-utils]` |2.0.2 
-|`./dynamic-plugins/dist/roadiehq-scaffolder-backend-module-utils-dynamic`
-
-
+%%TABLE_CONTENT_3%%
 |===
```
Comment (Gerry-Forde) [inline modules/dynamic-plugins/ref-community-plugins.adoc:19]: @nickboldt No output is generated for the community plugins.

Comment (nickboldt) [reply modules/dynamic-plugins/ref-community-plugins.adoc:19]: hmm. good catch. that's odd. will look later. BTW you'll get a faster response in slack than in GH where you have to wait hours for me to find the email notification under the 100 other email notifications I have from jira and gh. :D

File: modules/dynamic-plugins/ref-community-plugins.adoc [hunk 2]
```diff
@@ -10,267 +10,10 @@
 Details on how {company-name} provides support for bundled community dynamic plugins are available on the https://access.redhat.com/policy/developerhub-support-policy[Red Hat Developer Support Policy] page.
 ====
 
-{product-very-short} includes the following 50 community plugins:
+{product-very-short} includes the following %%COUNT_3%% community plugins:
```
Comment (themr0c) [inline modules/dynamic-plugins/ref-community-plugins.adoc:13]: %%COUNT_3%% is weird

Comment (Gerry-Forde) [reply modules/dynamic-plugins/ref-community-plugins.adoc:13]: @themr0c This script to generate plugins content is being reviewed by Nick.

**Review by Gerry-Forde — APPROVED** [review]
This needs to be merged so that we can run the content generation script locally and make any further content changes if necessary.

**Pull request merged**